app/views/amusement_parks/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="app">
+
+</div>
```

Comment:
Both this and the `new` erb files should have this as self-closing: `<div id='app' />`